app/controllers/volunteers_controller.rb

```diff
@@ -134,4 +144,22 @@ def update_volunteer_params
       .new(params)
       .without_active
   end
+
+  def volunteers_phone_number
+    authorize @volunteer
+    @volunteers_phone_number = @volunteer.phone_number
+  end
+
+  def send_sms_to(phone_number, body)
+    twilio = TwilioService.new(current_user.casa_org.twilio_api_key_sid, current_user.casa_org.twilio_api_key_secret, current_user.casa_org.twilio_account_sid)
+    req_params = {From: current_user.casa_org.twilio_phone_number, Body: body, To: phone_number}
+    twilio_res = twilio.send_sms(req_params)
+
+    # Error handling for spec test purposes
```

Comment:
It might be better to just leave `twilio.send_sms(req_params)` as the last line in the method so it is returned by default, and then assert on  `twilio_res.error_code` directly in spec